swagger.yaml

```diff
@@ -1,24 +1,32 @@
 swagger: "2.0"
 info:
-  description: "An API used to customise and submit queries (filters) against a given dataset. Once the filter job has been created it can then be updated with dimension-specific filters that apply to that dataset. Once all the required filters have been applied to the job, it is expected that the client will update the job state to indicate that processing can begin. Once processing has been started the job will be placed in a locked condition such that no further amendments can be done. Finally, the state will be marked as completed (or failed)."
+  description: "An API used to customise and submit queries (filters) against a given dataset. Once the filter blueprint has been created it can then be updated with dimension-specific filters that apply to that dataset. Once all the required filters have been applied to the blueprint, it is expected that the client will send a Put request with query parameter `submitted` set to `true` to indicate that processing can begin; this request will result in a filter output resource to be created, copying the data from the blueprint and setting a status of submitted. The response for this request will give a link to the created filter output document. Once processing has been started the filter output resource will be placed in a locked condition such that onlu authorised requests can update the resource. Finally, the state will be marked as completed with a complete list of downloads (or failed)."
```

Comment:
There a typo here.  `placed in a locked condition such that onlu authorised requests`

Reply:
Also i'm if wondering if a blueprint shouldn't have a state. The reason being `completed` will never happen, once created and submitted if I go back and edit the blueprint will it always be `submitted`?

Reply:
Yes I thought about this before and you are right the filter blueprint wont have a state; the filter output will still have a state but it will go from created to completed (there wont be a submitted state as this is just the creation of this resource).